packages/doc/src/components/Navigation/Navigation.jsx

```diff
@@ -240,7 +239,7 @@ const Navigation = ({ items, toggleMenu, opened, prefix }) => {
 
   return (
     <Wrapper opened={opened}>
-      <Nav>
+      <Nav aria-label="Contents navigation">
```

Comment:
```suggestion
      <Nav aria-label="Navigation content">
```

Reply:
As we discussed internally, I'm actually removing this whole aria-* attribute since nav itself already includes it and is accessible by default 🌻